modules/gitops-installing-argo-rollouts-cli-using-linux.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-installing-argo-rollouts-using-linux_{context}"]
+= Installing Argo Rollouts CLI using Linux
+
+For Linux, you can install Argo Rollouts CLI by performing the following steps.
+
+.Prerequisites
+
+* You have installed the {OCP} CLI (oc).
+
+.Procedure
+
+. Download the latest version of the Argo Rollouts CLI binary, `kubectl-argo-rollouts`, by running the following command using `curl`.
++
+.Example - Downloading the Argo Rollouts CLI binary
+[source,yaml]
+----
+curl -LO https://github.com/argoproj/argo-rollouts/releases/latest/download/kubectl-argo-rollouts-linux-amd64
+----
+
+. Ensure that the `kubectl-argo-rollouts` binary is executable by running the following command.
++
+.Example - Marking the `kubectl-argo-rollouts` binary as executable
+[source,yaml]
+----
+chmod +x ./kubectl-argo-rollouts-linux-amd64
+----
+
+. Move the `kubectl-argo-rollouts` binary to the system PATH by running the following command.
++
+.Example - Moving the `kubectl-argo-rollouts` binary to the system path
+[source,yaml]
+----
+sudo mv ./kubectl-argo-rollouts-linux-amd64 /usr/local/bin/kubectl-argo-rollouts
+----
+
+The Argo Rollouts CLI binary is installed.
+
+.Verification
+
+After you have completed the installation of the Argo Rollouts CLI plugin, verify the installation of the Argo Rollouts CLI plugin by running the following command:
+
+.Example - Verifying the Argo Rollouts version
+[source,yaml]
+----
+oc argo rollouts version
+----
+
+As a result, the following output is displayed:
+
+.Example - Output of the Argo Rollouts version command
+[source,yaml]
```

Comment:
```suggestion
+
.Example output
[source,terminal]
```

Reply:
this is how you add exmaple outputs, see: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output

Reply:
Fixed.